purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -237,7 +237,9 @@ class Purchases internal constructor(
          * being used
          */
         @JvmStatic
-        var platformInfo: PlatformInfo = PurchasesOrchestrator.platformInfo
+        var platformInfo: PlatformInfo
+            get() = PurchasesOrchestrator.platformInfo
+            set(value) { PurchasesOrchestrator.platformInfo = value }
```

Comment:
This was forgotten from #1144. Made the change here to avoid conflicts later

Reply:
I guess this doesn't really matter since we use these 2 only when configuring the SDK, and we use the value from `Purchases`, not `PurchasesOrchestrator`... We might be able to remove the ones in `PurchasesOrchestrator`. But for now, this seems more secure.